### PR TITLE
Describe annotations, delete URIType, add FontName, Border, and Margin

### PR DESCRIPTION
**Pull request by bernhardgr42**
Describe annotations and delete of URIType

### CODE REVIEW

File: vocabularies/PDF.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+          </Record>
+          <Record>
+            <PropertyValue Property="rel" String="describedby" />
+            <PropertyValue Property="href" String="https://github.com/sap/odata-vocabularies/blob/main/vocabularies/PDF.md" />
+          </Record>
+        </Collection>
+      </Annotation>
+
+      <Term Name="Features" Type="PDF.FeaturesType" Nullable="false" AppliesTo="EntityContainer">
+        <Annotation Term="Core.Description" String="..." />
```
Comment (ralfhandl) [inline vocabularies/PDF.xml:36]: A description would be nice

File: vocabularies/PDF.xml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        <Annotation Term="Core.LongDescription" String="..." />
```
Comment (ralfhandl) [inline vocabularies/PDF.xml:37]: `LongDescription` is optional and should differ from the `Description`.

File: vocabularies/PDF.xml [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      </Term>
+
+      <ComplexType Name="FeaturesType">
+        <Property Name="DocumentDescriptionReference" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.IsURL" Bool="true" />
+          <Annotation Term="Core.Description" String="Reference of the Service for the DocumentDescription" />
+          <Annotation Term="Core.LongDescription">
+            <String>Reference of the Service for the DocumentDescription</String>
+          </Annotation>
```
Comment (ralfhandl) [inline vocabularies/PDF.xml:46]:
```suggestion
```
Please remove `LongDescription` if it is identical to the `Description` or does not add information.

**Review by ralfhandl — CHANGES_REQUESTED** [review]

**Comment by ralfhandl** [comment]
@bernhardgr42 Thanks for contributing this! Please sign the [Contributor License Agreement](https://cla-assistant.io/SAP/odata-vocabularies?pullRequest=159) so we can accept your contribution.

**Review by bernhardgr42 — COMMENTED** [review]
* Remove URIType
* Add description to PDFFeatures
* Remove LongDescprition for some items.

**Comment by bernhardgr42** [comment]
* Remove URIType
* Add descrpition for PDFFeatures
* Remove LongDescprition for some items

**Review by bernhardgr42 — COMMENTED** [review]
r

**Review by bernhardgr42 — COMMENTED** [review]
Please review again.

**Review by ralfhandl — DISMISSED** [review]
Lgtm, @HeikoTheissen please double-check

File: vocabularies/PDF.xml [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+      <ComplexType Name="FeaturesType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="DocumentDescriptionReference" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.IsURL" Bool="true" />
+          <Annotation Term="Core.Description" String="Reference of the Service for the DocumentDescription" />
+        </Property>
+        <Property Name="DocumentDescriptionCollection" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Collection of DocumentDescription" />
+        </Property>
```
Comment (HeikoTheissen) [inline vocabularies/PDF.xml:52]: Is this just an entity set name? Or can it also be a collection-valued expression, e.g., an implicit entity set?

File: vocabularies/PDF.xml [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+          <Annotation Term="Core.LongDescription">
+            <String> Default result size for PDF documents. Used if a the $top has not been provided.</String>
+          </Annotation>
+        </Property>
+        <Property Name="ResultSizeMaximum" Type="Edm.Int32">
+          <Annotation Term="Core.Description" String="Maximum result size" />
+          <Annotation Term="Core.LongDescription">
+            <String>Max result size for PDF documents. Used if $top has been provided and $top > ResultSizeMaximum</String>
+          </Annotation>
+        </Property>
```
Comment (HeikoTheissen) [inline vocabularies/PDF.xml:91]: The client is forced to perform the paging by sending hundreds(?) of requests with suitable `$top` values and organize the hundreds(?) of downloaded PDF documents. This is unusable for customers who want to use PDF/A as an archive format.

File: vocabularies/PDF.xml (hunk 4 above)
Comment (bernhardgr42) [reply vocabularies/PDF.xml:52]: Only an EntitySet name

File: vocabularies/PDF.xml [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+      </Term>
+
+      <ComplexType Name="FeaturesType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="DocumentDescriptionReference" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.IsURL" Bool="true" />
+          <Annotation Term="Core.Description" String="Reference of the Service for the DocumentDescription" />
+        </Property>
+        <Property Name="DocumentDescriptionCollection" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Collection of DocumentDescription" />
```
Comment (HeikoTheissen) [inline vocabularies/PDF.xml:51]:
```suggestion
          <Annotation Term="Core.Description" String="Name of entity set containing the DocumentDescription" />
```

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

File: vocabularies/PDF.xml (hunk 5 above)
Comment (bernhardgr42) [reply vocabularies/PDF.xml:91]: This depends if the service likes to protect the server from a DoD attack. It is not a must have to switch on DefaultPaging (enabled in RAP).


This was discussed with S/4 and the default paging size for PDF's are higher than for normal data requests.

**Comment by HeikoTheissen** [comment]
> Lgtm, @HeikoTheissen please double-check

Fine, but someone else must approve.

**Review by ralfhandl — APPROVED** [review]

**Review by uhlmannm — APPROVED** [review]
I think the "Experimental" flag is important here and should not be removed before we have positive customer feedback on the end-to-end functionality.

**Pull request merged**